crates/bdk/Cargo.toml

```diff
@@ -13,6 +13,7 @@ edition = "2021"
 rust-version = "1.63"
 
 [dependencies]
+log = "^0.4"
```

Comment:
We shouldn't need this dependency, see #1222 for a bit of discussion about it.

Reply:
Good call. I did forked out a very old state of the bdk repo
Removed now.